client/i18n/locales/english/intro.json

```
@@ -881,6 +881,10 @@
           "This is one of the required projects to claim your certification."
         ]
       },
+      "learn-encapsulation-by-building-a-projectile-trajectory-calculator": {
+        "title": "Learn Encapsulation by Building a Projectile Trajectory Calculator",
+        "intro": ["", ""]
```

```suggestion
        "intro": [
          "Encapsulation is a core OOP principle based on writing code that limits direct access to data.",
          "In this project, you'll discover new concepts related to the encapsulation, such as getters, setters, and name mangling, and you'll use them together with what you already learn to create a program that calculates a projectile trajectory."
        ]
```
It's awful. Feel free to change it.